src/components/SearchFilters/SearchFilters.css

```diff
@@ -24,17 +18,19 @@
   }
 }
 
+.resultOrganization {
```

Comment:
Sounds company level stuff and about results instead of filters and sorting. 
How about: 'searchCustomization', 'queryParameters', or 'searchOptions'?